api/src/main/java/com/epam/pipeline/controller/datastorage/lifecycle/DataStorageLifecycleController.java

```diff
@@ -221,4 +225,88 @@ public Result<List<StorageLifecycleRuleExecution>> listStorageLifecyclePolicyRul
                 dataStorageLifecycleApiService.listStorageLifecyclePolicyRuleExecutions(
                         datastorageId, ruleId, path, status));
     }
+
+    @PostMapping(value = "/datastorage/{datastorageId}/lifecycle/restore")
+    @ResponseBody
```

Comment:
I missed this in the initial request - let's use `@RestController` to get rid of multi`@ResponseBody` annotations

Reply:
Fixed